domain/schema/controller.go

```diff
@@ -649,6 +650,21 @@ CREATE TABLE model_migration_minion_sync (
 );`)
 }
 
+func modelLastLogin() schema.Patch {
+	return schema.MakePatch(`
+CREATE TABLE model_last_login (
+    model_uuid TEXT NOT NULL,
+    user_uuid TEXT NOT NULL,
```

Comment:
We can remove this https://github.com/juju/juju/blob/edadf36e3f6b82b023eecf8a4c37a5bb2e2cea3e/domain/schema/controller.go#L763, because you can answer the same question with this table.

`SELECT user_uuid, MAX(time) FROM model_last_login WHERE user_uuid=?`